src/content/guides/build-performance.md

```diff
@@ -152,6 +155,57 @@ new CommonsChunkPlugin({
 })
 ```
 
+### Avoid Extra Optimization Steps
+
+webpack does extra algorithmic work to optimize the output for size and load performance. These are usually not needed during iterative development:
+
+```js
+optimization: {
+  removeAvailableModules: false,
+  removeEmptyChunks: false,
+  splitChunks: false,
+}
+```
+
+### Output Without Path Info
+
+webpack has the ability to generate path info in the output bundle. However, this puts inconsequential garbage collection pressure on projects that bundle thousands of modules. Turn this off in the `options.output.pathinfo` setting:
+
+```js
+{
+  output: {
+    pathinfo: false
+  }
+}
+```
+
+### Node.js Version
+
+There has been a [performance regression](https://github.com/nodejs/node/issues/19769) in the latest stable versions of node.js with regard to the ES2015 Map and Set implementation. A fix has been merged in master, but a release has yet to be made. In the meantime, to get the most out of incremental build speeds, try to stick with version 8.9.x (problem exists between 8.9.10 - 9.11.1). webpack has moved to using those ES2015 data structures liberally, and it will improve the initial build times as well.
+
+### Typescript
```

Comment:
~Typescript~ TypeScript :)

Reply:
I've been typing it wrong the whole time!!

Reply:
Same with JavaScript. :P

Reply:
You've been "typing" TypeScript wrong. That's a good one.